src/main/java/uk/nhs/digital/nhsconnect/nhais/service/TimestampService.java

```diff
@@ -1,15 +1,21 @@
 package uk.nhs.digital.nhsconnect.nhais.service;
 
 import org.springframework.stereotype.Component;
-import uk.nhs.digital.nhsconnect.nhais.utils.TimestampUtils;
 
-import java.time.ZonedDateTime;
+import java.time.Instant;
+import java.time.ZoneOffset;
+import java.time.format.DateTimeFormatter;
 
 @Component
 public class TimestampService {
-
-    public ZonedDateTime getCurrentTimestamp() {
-        return TimestampUtils.getCurrentDateTimeAsUTC();
+    public Instant getCurrentTimestamp() {
+        var now = Instant.now();
```

Comment:
Consider using `Clock` class (for fixing time in tests):
````java
    public Instant getCurrentTimestamp(Clock clock) {
        var now = Instant.now(clock);
        return now.minusNanos(now.getNano());
    }

    public Instant getCurrentTimestamp() {
        return getCurrentTimestamp(Clock.systemUTC());
    }
````

Reply:
I think it's not needed as the `getCurrentTimestamp` itself is in a class that will be injected, so it can be mocked easily